ISISScientificComputing/autoreduce
# Fixes configuring experiment variables submission after toggling from run variables

This is based on #1042 and should be merged after

### Summary of work
Removes Configure new Runs form toggle - now two separate forms are generated from the templating. 

Fixes submission for experiments

### How to test your work
Follow the PR quickstart for initial steps to start AR locally: https://github.com/ISISScientificComputing/autoreduce/wiki/PR-testing-quickstart

Then go to the instrument runs, and Configure New Runs. Change between types (run range & experiment vars) and submit some for each. They should show up in the "See instrument variables" page

### Additional comments


Fixes #1154


**Before merging ensure the release notes have been updated**
